crates/api_models/src/payment_methods.rs

```diff
@@ -6,76 +6,153 @@ use utoipa::ToSchema;
 
 use crate::enums as api_enums;
 
-#[derive(Debug, serde::Deserialize, serde::Serialize, Clone)]
+#[derive(Debug, serde::Deserialize, serde::Serialize, Clone, ToSchema)]
 #[serde(deny_unknown_fields)]
 pub struct CreatePaymentMethod {
+    /// The type of payment method use for the payment.
+    #[schema(value_type = PaymentMethodType,example = "card")]
     pub payment_method: api_enums::PaymentMethodType,
+    /// This is a sub-category of payment method.
+    #[schema(value_type = Option<PaymentMethodSubType>,example = "credit_card")]
     pub payment_method_type: Option<api_enums::PaymentMethodSubType>,
+    /// The name of the bank/ provider issuing the payment method to the end user
+    #[schema(example = "Citibank")]
     pub payment_method_issuer: Option<String>,
+    /// A standard code representing the issuer of payment method
+    #[schema(value_type = Option<PaymentMethodIssuerCode>,example = "JP_APPLEPAY")]
     pub payment_method_issuer_code: Option<api_enums::PaymentMethodIssuerCode>,
```

Comment:
We've added a `#[serde(rename_all = "snake_case")]` annotation to `PaymentMethodIssuerCode` struct. So the example should be snake case instead of screaming snake case.